mix.exs

```diff
@@ -62,7 +62,7 @@ defmodule EctoJobScheduler.MixProject do
       {:excoveralls, "~> 0.10", only: :test},
       {:ex_doc, "~> 0.14", only: :dev, runtime: false},
       {:dialyxir, "~> 1.0", only: [:dev], runtime: false},
-      {:new_relic_agent, "~> 1.19", optional: true},
+      {:new_relic_agent, "~> 1.24.1", optional: true},
```

Comment:
why not use the latest version? 1.24.3

Reply:
No reason, my fault. I updated the commit with the last version